app/assets/stylesheets/base/_forms.scss

```diff
@@ -0,0 +1,11 @@
+input {
+  width: 100%;
+  padding: rem(5px);
+  border: 0;
+  border-bottom: rem(1px) solid $primary;
```

Comment:
```suggestion
  border-bottom: 1px solid $primary;
```
This one also

Reply:
Updated in https://github.com/aetheryna/google-scraper-ruby/pull/32/commits/c50b3c3e72925a76b1b328a88f4d9229fbb60f9b 😎